docs/css/main.css.map

```diff
@@ -1 +0,0 @@
U,gC2CldX,kBACA,cACA,M3CmsB4B,Q2ClsB5B,qBACA,gBACA,uBACA,sBACA,WACA,gBACA,YACA,gCACA,kCACA,4CACA,eAEA,+EACC,iB3CksB8B,QA5I/B,+EATC,0BAWA,mBAID,uGACC,S2CxjBD,+E3C0cC,KAdY,IAeZ,KAdU,OAaV,MAdY,KAeZ,MAdU,O2C1bV,WACA,kBACA,MACA,SACA,kRAGA,4BACA,6BAKA,wLACC,QACA,YAIF,yIACC,kB3C0pB2B,Q2CzpB3B,iBAEA,qJACC,2dAUA,oUACC,cAMF,mJACC,6gCAQA,gUACC,YAKH,wRAEC,M3C0oB+B,K2CzoB/B,oB3C0pBkC,QAvKnC,gTAZC,0BAcA,mBAID,gWACC,S2CtfA,gTACC,iB3CipBiC,Q2C9oBlC,gTACC,kRAKD,wbACC,6gCAMD,oaACC,kB3ConB8B,Q2ClnB9B,4bACC,2dAWJ,+B3C8fC,eACA,kBACA,YALa,a2Czfb,cACA,mBACA,M3C0lB+B,Q2CxlB/B,4DACC,M3C8mBkC,Q4CxxBpC,sCCiBA,yBACC,kBACA,cACA,SACA,UACA,gBAEA,0DACC,kBACA,MACA,OACA,YACA,WACA,SAGD,uDACC,sBAGD,sDACC,mBAGD,6B7CkeC,WAdY,KAeZ,WAdU,K6C5cZ,+DAGC,eACA,YAGD,2E7CkdE,WAdY,KAeZ,WAdU,K8C9fZ,4BCSA,aACC,WACC,6BACA,0RCKF,WhDgpBC,eACA,eACA,YALa,IAnJZ,QAdY,kBAeZ,QAdU,8BAaV,OAdY,KAeZ,OAdU,SgDreX,kBACA,sBACA,qBACA,gBACA,yBACA,kBACA,iBACA,iBhDotB4B,KgDntB5B,sBACA,chDswBkB,IgDrwBlB,eACA,iRAIA,4BACA,iCACA,2BACA,sBhDukBA,iBATC,0BAWA,mBAID,6BACC,SgD1kBD,iBACC,gBAGD,qBACC,mBACA,iEACA,WAGD,oBACC,mBACA,iEACA,WAID,sDAEC,yBACA,ahD4tBiB,QgDztBlB,4EAEC,yBACA,ahDstBiB,QgDntBlB,wEAEC,yBACA,ahDitBiB,QgD5sBjB,4CACC,aAIF,uBACC,aAGD,gChD8aC,cAdY,IAeZ,cAdU,QgD5ZX,2BACC,ahDqqB+B,KAtJhC,iCAZC,0BAcA,mBAID,6CACC,SgDlhBD,4BACC,WAQF,0BACC,kBACA,uBAGD,iDhDsZE,cAdY,KAeZ,cAdU,KgDrYZ,yCACC,qBhDiZC,cAdY,KAeZ,cAdU,SiD9fZ,+BCSA,aACC,oBACC,4BCKF,cnDipBC,eACA,kBACA,YALa,EmD5oBb,YnDsrBS,wImDrrBT,MnD8tB4B,QmD3tB7B,oBnD4jBC,kBACA,UACA,WACA,UACA,YACA,gBACA,sBACA,SmDhkBA,qDnDglBC,0BA9FA,IAdY,KAeZ,IAdU,KAaV,KAdY,KAeZ,KAdU,KAaV,QAdY,KAeZ,QAdU,OmD/dV,UACA,YACA,SACA,iBACA,kBACA,WACA,MnD+sB2B,KmD9sB3B,iBnD4sB2B,QmD3sB3B,0BACA,yBACA,YAGD,6DAEC,qBACA,MnDqsB2B,KoDpvB7B,0BCSA,aACC,SACC,sBAED,eACC,6BACA,iBACC,uBCEH,StD+oBC,eACA,eACA,YALa,IsD1oBb,cACA,SACA,UACA,YtDirBS,wIsDhrBT,MtDytB4B,QsDvtB5B,uBACC,MtD6uB+B,KsDzuBjC,ctD4eE,OANY,YAOZ,OANU,mBAKV,QAdY,MAeZ,QAdU,SsD5dX,yBACA,ctDqwBkB,IsDpwBlB,qBAEA,yCtDqeC,OAdY,OAeZ,OAdU,UAaV,QAdY,MAeZ,QAdU,SsDpdV,MtDwsB2B,QsDvsB3B,qBACA,ctD4vBiB,IsD1vBjB,qDACC,MtDksB0B,QsDjsB1B,qBACA,iBtD6sB6B,QA5I/B,qDATC,0BAWA,mBAID,6EACC,SsDnkBA,qDACC,oBAKH,4BACC,atD4sBgC,QsD1sBhC,qEAEC,MtDwsB+B,QsDtsB/B,iFACC,MtDosB8B,KsDnsB9B,iBtDgtBiC,QAnKnC,iFAZC,0BAcA,mBAID,yGACC,SuDznBF,iCCSA,aACC,eACC,6BAEA,yBACC,yBACA,iCACA,iCAGD,wBACC,yBACA,iCACA,kCCUH,eACC,sBzDgoBA,eACA,eACA,YALa,EAnJZ,QAdY,SAeZ,QAdU,YAaV,OAdY,KAeZ,OAdU,SyDzdX,yBACA,iBzD8sB4B,KyD7sB5B,YzDiqBS,wIyDhqBT,MzDysB4B,QyDxsB5B,czD+vBkB,IyD9vBlB,sBACA,gBAGA,kEAEC,yBACA,azD8uBiB,QyD3uBlB,gGAEC,yBACA,azDwuBiB,QyD5tBlB,4FAEC,yBACA,azD0tBiB,QyDttBlB,yBACC,mBACA,iEACA,WAGD,wBACC,mBACA,iEACA,WzD+hBD,qBATC,0BAWA,mBAID,iCACC,SyDjiBD,qBACC,gBACA,iBzD0pB2B,KyDvpB5B,mCACC,azD0qB+B,KyDxqB/B,6CACC,iEACA,WAGD,4CACC,iEACA,WzD2gBF,yCAZC,0BAcA,mBAID,qDACC,SyDxgBD,oCACC,cACA,WACA,sBASD,qCACC,YzD2kBkB,sFyD1kBlB,mBA2CF,uBzDufC,eACA,eACA,YALa,IyDlfb,YACA,eACA,eCtKD,KACC,SAGD,sB1D+fE,WAdY,KAeZ,WAdU,KAaV,QANY,OAOZ,QANU,O0DrfZ,K1D0fE,WAdY,KAeZ,WAdU,K0DzeZ,K1DsfE,WAdY,KAeZ,WAdU","file":"main.css"}
```

Comment:
We should probably keep the map file. Have a look at what they do and how to make one using node-sass.